superset/models.py

```diff
@@ -1410,7 +1421,8 @@ def visit_column(element, compiler, **kw):
 
         qry = qry.limit(row_limit)
 
-        if is_timeseries and timeseries_limit and groupby:
+        # Some SQL dialects use timeslot grouping inline with other dimensions
+        if is_timeseries and timeseries_limit and groupby and not time_groupby_inline:
```

Comment:
I don't think I understand this statement and I know the Vertica optimizer intimately.